install/opensuse-installation-guide.md

```diff
@@ -0,0 +1,32 @@
+# Install Kata Containers on openSUSE Leap
+
+> **Notes:**
+>
+> - Kata Containers packages are available for [openSUSE\*](https://www.opensuse.org/)
+>   version 43.2 (currently `x86_64` only).
+>
+> - If you are installing on a system that already has Clear Containers or `runv` installed,
+>   first read [the upgrading document](../Upgrading.md).
+>
+> - If you do not want to copy or type all these instructions by hand, you can use the
+>   [`kata-manager`](https://github.com/kata-containers/tests/blob/master/cmd/kata-manager/kata-manager.sh)
+>   script to install the packaged system including your chosen container
+>   manager. Alternatively, you can generate a runnable shell script from
+>   individual documents using the
+>   [`kata-doc-to-script`](https://github.com/kata-containers/tests/blob/master/.ci/kata-doc-to-script.sh) script.
+
+1. Install the Kata Containers components with the following commands:
+
+   > **Note:** This installation channel is not secure since the repository currently
+   > redirects download URLs to `http`.
+
+   ```bash
+   $ source /etc/os-release
+   $ sudo -E VERSION_ID=$VERSION_ID zypper addrepo "https://download.opensuse.org/repositories/home:/katacontainers:/release/openSUSE_Leap_${VERSION_ID}/home:katacontainers:release.repo"
```

Comment:
Until OBS/Suse resolve the https -> http issue for OpenSuSE+SLES (https://github.com/kata-containers/documentation/pull/83#issuecomment-412438187), this URL needs to be http imho.